data_steward/analytics/tools/dq_issue_tracker/class_definitions.py

```diff
@@ -257,6 +291,21 @@ def find_failing_metrics(self):
                     thresholds['route_success_min']:
                 failing_metrics.append(ingredient_integration_obj)
 
+        for date_disparity_obj in self.date_datetime_disparity:
+            if date_disparity_obj.value > \
+                    thresholds['date_datetime_disparity_max']:
+                failing_metrics.append(date_disparity_obj)
```

Comment:
Not sure if it's necessary to split this line.

Reply:
can change this in EDQ-440 as well